block-storage/src/main/scala/coop/rchain/blockstorage/dag/BlockDagRepresentationSyntax.scala

```diff
@@ -38,6 +38,37 @@ final class BlockDagRepresentationOps[F[_]: Sync](
     dag.lookup(hash) >>= (_.liftTo(BlockDagInconsistencyError(errMsg)))
   }
 
+  def lookupUnsafe(hashes: Seq[BlockHash])(
+      implicit line: sourcecode.Line,
+      file: sourcecode.File,
+      enclosing: sourcecode.Enclosing,
```

Comment:
Source info is not used here and it can be removed.

But even more, as mentioned in the top comment, this whole _source_ parameters are not needed anymore because monix@v3 has async stack trace so we have source reference info on errors.

Reply:
@vb-cs maybe you can take this task to remove _source_ parameter?

I want to point you to this pattern of extension methods written as class methods. In this situation class must accept only one parameter, extend AnyVal for better performance etc. 

In Scala 3 this pattern will have syntax representation. I didn't test this but it should be something like this.
``` scala
extension [F[_]: Sync] (dag: BlockDagRepresentation[F]):
  def lookupUnsafe:
    def errMsg = s"DAG storage is missing hash ${PrettyPrinter.buildString(hash)}"
    dag.lookup(hash) >>= (_.liftTo(BlockDagInconsistencyError(errMsg)))
```